google/cloud/spanner/internal/session_pool.cc

```diff
@@ -29,6 +29,17 @@ namespace internal {
 
 namespace spanner_proto = ::google::spanner::v1;
 
+std::shared_ptr<SessionPool> MakeSessionPool(
+    Database db, std::vector<std::shared_ptr<SpannerStub>> stubs,
+    SessionPoolOptions options, std::unique_ptr<RetryPolicy> retry_policy,
+    std::unique_ptr<BackoffPolicy> backoff_policy) {
+  auto pool = std::make_shared<SessionPool>(
+      std::move(db), std::move(stubs), std::move(options),
+      std::move(retry_policy), std::move(backoff_policy));
+  pool->Initialize();
```

Comment:
Do we need a breadcrumb to remind us why this can't be in the ctor, given that the usual reason (possibility of failure) don't apply.

Reply:
Done.